src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -333,6 +350,11 @@ public Builder subscriber(UnleashSubscriber unleashSubscriber) {
             return this;
         }
 
+        public Builder fallbackStrategy(@Nullable Strategy fallbackStrategy) {
```

Comment:
Generally it seems a bit strange to accept a "null" strategy here. It is not totally clear to me the purpose of that, and is it clear to the users that you then will get the "UNKNOWN" strategy?

Reply:
I don't have a good use case for null here except as error handling/recovery